### PR TITLE
BSIP59: Adjustment of MSSR and MCR through voting

### PR DESCRIPTION
**Pull request by bitcrab**
create bsip59

### CODE REVIEW

File: bsip-0059.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+
+
+    BSIP: 59
+    Title: Adjustment of MSSR and MCR through voting
+    Authors: Jerry Liu <bitcrab@qq.com>
+    Status: Draft
+    Type: Protocol
```
Comment (pmconrad) [inline bsip-0059.md]: Please change to `Informational`

File: bsip-0059.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+3.bitCNY premium > (MSSR-1) 
+In this phase the margin call orders will stay there without being eaten, global settlement price will not drop while the feed price drop, risk accumulate.
+
+4.market price drop below the global settlement price
+While global settlement protection measure is implemented, global settlement will not happen, feed price will stop to drop following the market price, if BTS price continue to drop, bitCNY will devalue.
+
+It’s best for bitCNY to be always in phase 2, the system should always avoid bitCNY to fall in phase 4 in comparatively high BTS price.
+If bitCNY fall in phase 3 for long time, it will be good to reduce the premium by reducing MSSR.
+Decreasing of MCR will help to supply more bitCNY and help bitCNY to jump from phase 3 to phase
```
Comment (pmconrad) [inline bsip-0059.md]: Please complete this sentence.

File: bsip-0059.md (hunk 1 above)
Comment (bitcrab) [reply bsip-0059.md]: done.

File: bsip-0059.md (hunk 2 above)
Comment (bitcrab) [reply bsip-0059.md]: completed, thanks.

File: bsip-0059.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
+
+
+    BSIP: 59
+    Title: Adjustment of MSSR and MCR through voting
+    Authors: Jerry Liu <bitcrab@qq.com>
+    Status: Draft
+    Type: Informational
+    Created: 2009-02-15
```
Comment (xeroc) [inline bsip-0059.md]: I guess you meant 2019

Comment (bitcrab) [reply bsip-0059.md]: changed, thanks.

**Comment by ryanRfox** [comment]
Thank you @bitcrab for correcting the date. I will dismiss the pending review for @xeroc and proceed to merge.

**Pull request merged**